src/probnum/filtsmooth/statespace/transition.py

```diff
@@ -149,6 +154,8 @@ def transition_rv(
         linearise_at :
             For approximate transitions , for instance ContinuousEKFComponent,
             this argument overloads the state at which the Jacobian is computed.
+        diffusion :
+            Diffusion (or diffusion estimate) for this transition
```

Comment:
maybe mention that this field is usually used to update diffusions with calibrated versions thereof?